examples/mergable-ingress-types/README.md

```diff
@@ -0,0 +1,105 @@
+# Mergeable Ingress Types Support
+
+You can configure NGINX to have support for using multiple Ingress Resources which apply to a common host. This allows
+for easier management when using a large amount of paths.
+
+## Syntax and Rules
+
+A Master is declared using `nginx.org/mergible-ingress-type: master`. A Master will process all configurations at the
+host level, which includes the TLS configuration, and any annotations which will be applied for the complete host. There
+can only be one ingress resource on a unique host that contains the master value. Paths cannot be part of the
+ingress resource.
+
+A Minion is declared using `nginx.org/mergible-ingress-type: minion`. A Minion will be used to append different
+locations to an ingress resource with the Master value. Only annotations which contain the serviceName key can be used
```

Comment:
Could you change the part about annotations to something like "for now, the annotations of minions are replaced with the annotations of their master"

Reply:
Done